icu4c/APIChangeReport.md

```diff
@@ -23,11 +23,49 @@ Removed from ICU 75
   
 | File | API | ICU 75 | ICU 76 |
 |---|---|---|---|
+| messageformat2_data_model.h | Builder&amp; icu::message2::MFDataModel::Builder::addUnsupportedStatement(UnsupportedStatement&amp;&amp;, UErrorCode&amp;) |  InternalICU 75 | (missing)
```

Comment:
@catamorphism @mihnita @echeran There are a lot of MF2 changes here, and below for Java -- both compared with what the API change report showed just a couple of weeks ago. Are these all covered by API proposals?

Reply:
Yes, this is due to https://github.com/unicode-org/icu/pull/3177 (Java) and https://github.com/unicode-org/icu/pull/3193 (C++). This removes unsupported expressions/statements from MF2, which includes removing some methods/classes from the data model. I followed Mihai's lead in not submitting a design doc. Should I have submitted one?

Reply:
API changes should always be proposed via icu-design email and a section in the API proposal doc.
Even for tech preview, it doesn't make sense to publicly propose the initial API and then silently change parts of it.

Reply:
Got it.